### ISSUE TITLE
RFE: show whether config was updated or not

### ISSUE THREAD

**Issue by xsuchy** [pixelb/crudini#21]
It would be nice if crudini can print whether it really altered the config. Eg:
  changed: file foo has been successfully altered
or
  ok: no change needed
when the var/config is already set or deleted.

This would allow better usage in ansible playbooks. Right now it is not possible if command actually changed something or not.

**Comment by pixelb** [comment]
Would the ansible playbooks process the feedback, or is this for logging?

In either case a --verbose option may work here to produce output like:

```created: file, section, new param, new value
created: new file, new section
updated: file, section param value
deleted: file section param value```
deleted: file section```

If we were adding new/old values to the --verbose output,
they'd have to be escaped as can be multiline etc.

**Comment by xsuchy** [comment]
--verbose would be nice

It does not need to be verbose too much (e.g. what actually was changed). I'm interested in True/False state: changed or not. Because if the config was changed I need to reload service. When there was no change I do not need to restart the service.

Ansible can parse output and detect the change based on return code or the command output:
http://docs.ansible.com/playbooks_error_handling.html#overriding-the-changed-result

So I can do something like:
- command: crudini --set /etc/nova/nova.conf DEFAULT  foo
  register: command_result
  changed_when: "'success' in command_result.stdout"

**Comment by pixelb** [comment]
OK I'll probably do something like the following on stderr with --verbose:

    changed: ....
    unchanged: ...

Then that can be easily matched with the re: ^changed

**Comment by pixelb** [comment]
Fixed with commit 7696072a

**Comment by pixelb** [comment]
Tweaked this to commit b43b6aa9

**Comment by pixelb** [comment]
Fixed with commit 9852e616
